### PR TITLE
KAFKA-17583 Improve the ConfigCommand documentation to show how to set `key=[val1,val2]`

### PR DESCRIPTION
**Pull request by Linsiyuan9**
…pact`

*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by Linsiyuan9** [comment]
I think we just need to boot the way square brackets are configured

**Comment by frankvicky** [comment]
Hi @Linsiyuan9 
Could you please rebase PR to apply new CI flow ?
Thank you

**Comment by Linsiyuan9** [comment]
hi @frankvicky ,I'm new to the community. What should I do?

**Comment by frankvicky** [comment]
Hi @Linsiyuan9 
All you need to do is rebase your PR on to trunk. 😸

**Comment by Linsiyuan9** [comment]
HI @frankvicky , https://github.com/apache/kafka/pull/17428

**Comment by frankvicky** [comment]
Hi @Linsiyuan9,
You don’t need to open a new PR. You should use your original branch (`KAFKA-17583`), rebase it onto trunk, and push it again.
In this way, you don't need to open a duplicate PR.

**Comment by Linsiyuan9** [comment]
Hello @frankvicky , because my error operation uses merge, it affects the linear display of the commit history

**Comment by Linsiyuan9** [comment]
hi @frankvicky,I'm sorry about the rush 🤐 .
I think this should be commit like this: https://github.com/apache/kafka/pull/17429😥

**Comment by Linsiyuan9** [comment]
https://github.com/apache/kafka/pull/17429

**Comment by TaiJuWu** [comment]
> Hello @frankvicky , because my error operation uses merge, it affects the linear display of the commit history

The policy of Kafka will squash commits when merge so it is allowed to merge trunk with merge command.

Please close other PRs, thanks a lot.

**Comment by Linsiyuan9** [comment]
@TaiJuWu ,Thank you for your answers, which will help me continue to contribute to the kafka community👍

File: core/src/main/scala/kafka/admin/ConfigCommand.scala [hunk 1]
```diff
@@ -299,7 +299,7 @@ object ConfigCommand extends Logging {
       val configsToBeAdded = opts.options.valueOf(opts.addConfig)
         .split("," + pattern)
         .map(_.split("""\s*=\s*""" + pattern, -1))
-      require(configsToBeAdded.forall(config => config.length == 2), "Invalid entity config: all configs to be added must be in the format \"key=val\".")
+      require(configsToBeAdded.forall(config => config.length == 2), "Invalid entity config: all configs to be added must be in the format of \"key=val\". If val is in the format symbol 'val1,val2', you can use \"key=[val1,val2]\".")
```
Comment (AndrewJSchofield) [inline core/src/main/scala/kafka/admin/ConfigCommand.scala]: I suggest the following: `"Invalid entity config: all configs to be added must be in the format \"key=val\" or  \"key=[val1,val2]\" to group values which contain commas."`

**Review by AndrewJSchofield — CHANGES_REQUESTED** [review]
Thanks for the PR. I think you are correct that the script works for a list of cleanup.policy values provided that they are supplied in `[]`. Just updating the error message is sufficient.

File: core/src/main/scala/kafka/admin/ConfigCommand.scala (hunk 1 above)
Comment (Linsiyuan9) [reply core/src/main/scala/kafka/admin/ConfigCommand.scala]: Yes, I have reproduced the commit code

**Review by AndrewJSchofield — APPROVED** [review]

**Review by chia7712 — APPROVED** [review]

**Pull request merged**